test/integration/testdata/busybox-mount-test.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   restartPolicy: Never
   containers:
     - name: mount-munger
-      image: busybox:1.28.4-glibc
+      image: gcr.io/k8s-minikube/busybox:1.28.4-glibc
```

Comment:
did we push this tag to GCR as well?

Reply:
nevermind, checked myself and it's there

Reply:
Yup, I pushed `latest`, `1.28` & `1.28.4-glibc`